cli-tests/tests/flow/all-operators.spec.ts

```diff
@@ -43,39 +43,21 @@ test.describe('Everest CLI install', async () => {
       });
     };
 
-    await test.step('run everest install command', async () => {
-      const out = await cli.everestExecSkipWizard(
-        `install --namespaces=everest-all -v`,
-      );
-
-      await out.assertSuccess();
-      await out.outErrContainsNormalizedMany([
-        'percona-xtradb-cluster-operator operator has been installed',
-        'percona-server-mongodb-operator operator has been installed',
-        'percona-postgresql-operator operator has been installed',
-        'everest-operator operator has been installed',
-      ]);
-    });
-    await page.waitForTimeout(10_000);
-    await verifyClusterResources();
-
```

Comment:
It seemed redundant to run the CLI tests twice for verbose logs and pretty logs. I think its just redundant and doesn't provide any real value, so removed them. Let me know if you think otherwise